package.json

```diff
@@ -137,6 +137,8 @@
     "cookie-session": "^2.1.0",
     "cypress": "^13.16.0",
     "cypress-axe": "^1.5.0",
+    "cypress-image-diff-html-report": "^2.2.0",
```

Comment:
What's the HTML report for? presumably just for seeing the results in the pipeline right?

Reply:
Oh good question, I didn't actually set that up properly. Details are here: https://github.com/kien-ht/cypress-image-diff-html-report?tab=readme-ov-file#overview